_search-plugins/sql/endpoints.md

```diff
@@ -17,20 +17,6 @@ other parameters passed to plugin for other functionality such as
 prepared statement. And also the explain endpoint is used very often for
 query translation and troubleshooting.
```

Comment:
To send a query to the SQL plugin you can either pass it as a query parameter to the HTTP GET request or pass it in the request body of the HTTP POST request. We recommend using a POST request because then the query is not limited in length and you can pass other parameters, such as a prepared statement, to the plugin. You can use the `_explain` endpoint for query translation and troubleshooting.